index.html

```diff
@@ -4158,29 +3834,10 @@ <h1>DID Methods</h1>
             <a href="https://github.com/unik-name/did-method-spec/blob/main/did-uns/UNS-DID-Specification.md">UNS DID Method</a>
           </td>
         </tr>
-        <tr>
-          <td>
-            did:uport:
-          </td>
-          <td>
-            DEPRECATED
```

Comment:
I opened https://github.com/w3c/did-spec-registries/issues/355 to address this issue.